src/java.rs

```diff
@@ -223,7 +237,7 @@ mod auto {
             //   public java.util.Date(int, int, int);
             //   public java.util.Date(int, int, int, int, int);
             //   public java.util.Date(int, int, int, int, int, int);
-            //   public java.util.Date(java.lang.String);
+            //  public java.util.Date(java.lang.String);
```

Comment:
```suggestion
            //   public java.util.Date(java.lang.String);
```

this does not look intentional